git_machete/cmd.py

```diff
@@ -1319,10 +1339,12 @@ def get_combined_remote_sync_status(b):
 # Reflog magic
 
 
-reflogs_cached = None
+REFLOG = Tuple[str, str]
+
+reflogs_cached: Optional[Dict[str, Optional[REFLOG]]] = None
```

Comment:
Shouldn't it rather be `Optional[Dict[str, List[REFLOG_ENTRY]]]`? This would explain the `type: ignore`s below

Reply:
Hmmm.... but the dict can return None in case of `reflog_cached.get(absent_key)` :c

Reply:
Okay... then `Optional[Dict[str, Optional[List[REFLOG_ENTRY]]]]`?